packages/grid/src/GridRenderer.js

```diff
@@ -1011,7 +1011,7 @@ class GridRenderer {
   }
 
   drawTreeMarker(context, state, columnX, rowY, treeBox, color, isExpanded) {
-    const [x1, y1, x2, y2] = treeBox;
+    const { x1, y1, x2, y2 } = treeBox;
```

Comment:
This should still be an array as far as I can tell.

Reply:
I changed this to an object everywhere, to the new `BoxCoordinates` type. I figured that was more descriptive than an array, and there was no real reason for it to be an array anyway.